java/pom.xml

```diff
@@ -74,20 +74,20 @@
         <commons.io.version>2.11.0</commons.io.version>
         <commons.text.version>1.10.0</commons.text.version>
         <commons.lang3.version>3.12.0</commons.lang3.version>
-        <debezium.version>2.4.2.Final</debezium.version>
+        <debezium.version>2.6.2.Final</debezium.version>
         <jackson.version>2.13.5</jackson.version>
         <spark_sql.version>3.3.1</spark_sql.version>
-        <hadoop.version>3.3.3</hadoop.version>
+        <hadoop.version>3.4.0</hadoop.version>
         <elasticsearch.version>7.17.19</elasticsearch.version>
-        <datastax.version>4.15.0</datastax.version>
-        <flink.version>1.18.0</flink.version>
+        <datastax.version>4.17.0</datastax.version>
+        <flink.version>1.19.1</flink.version>
```

Comment:
It seems that this upgrade on flink version causes compilation error. Our current mocked flink api is based on 1.18.0. If there is no special reason, we don't plan to upgrade it yet.

Reply:
got it, flink version has been downgraded to 1.8.0, 
please take a look again, thanks.